manage-mysql-user.py

```diff
@@ -122,20 +150,92 @@ def test_connection(username, password_parameter):
         return True
 
 
+def validate_event(event):
+    is_valid = True
+
+    if "mysql_user_username" not in event.keys():
+        logger.error(f"Invalid event: 'mysql_user_username' must be set")
+        is_valid = False
```

Comment:
maybe something like this would be more maintainable:
https://github.com/dwp/configure-confluent-kafka-consumer/blob/master/main.py#L130...L141
Is there only one required event key?

Reply:
The reason for doing it this way is two special cases of mutually exclusive arguments:
1. `mysql_user_password_parameter_name` vs `mysql_user_password_secret_name` in the event (aka input JSON)
and
2. `RDS_MASTER_PASSWORD_SECRET_NAME` vs `RDS_MASTER_PASSWORD_PARAMETER_NAME` in envvars.

Also, both of those aren't command line arguments and using argparser would be awkward even if not for mutual exclusivity.